index.js

```diff
@@ -74,6 +77,13 @@ module.exports = {
 			allowTemplateLiterals: true
 		}],
 		'semi': ['error', 'always'],
+		'space-before-blocks': ['error', {
+			functions: 'never',
+			keywords: 'never',
+			classes: 'always'
```

Comment:
As I [noted in the Slack thread](https://s.slack.com/archives/C64D4D78D/p1615185839114500?thread_ts=1615149358.112400&cid=C64D4D78D), I don't think this configuration is very conventional. It will likely cause friction and require a retrain for many new contributors. It would be better to enforce conventional whitespace or to not enforce.
```suggestion
			functions: 'always',
			keywords: 'always',
			classes: 'always'
```

Reply:
✅ in https://github.com/particle-iot/eslint-config-particle/pull/8/commits/3e0354d6beae6d5077a2544e712f63465fde459a